### PR TITLE
add blockchain cli

### PR DESCRIPTION
**Pull request by ac0stas**
command-line interface with functions for blockchain operations

### CODE REVIEW

File: cli/cli.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+	}
+	optionsAll ["getbalance"] = Option{
+		Text: "Get balance",
+		Handler: getBalance,
+	}
+	optionsAll ["printchain"] = Option{
+		Text: "Print chain",
+		Handler: func(string) {
+			printChain()
+		},
```
Comment (benja-iag) [inline cli/cli.go]: change this for

```suggestion
		Handler: printChain,
```

File: cli/cli.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	}
+	optionsAll ["searchblock"] = Option{
+		Text: "Search block by hash",
+		Handler: searchBlockByHash,
+	}
```
Comment (benja-iag) [inline cli/cli.go]: add the 'send' option

File: cli/cli.go [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+		}
+
+		//optionsAll[action].Handler = func(string) {
+		//	send(*sendFrom, *sendTo, *sendAmount)
+		//}
+		// Obtén los argumentos para el comando `send` de la variable `flag.Args()`.
+		sendArgs := flag.Args()[2:]
+
+		// Llama a la función `send()` con los argumentos obtenidos.
+		optionsAll[action].Handler(sendArgs[0])
```
Comment (benja-iag) [inline cli/cli.go]: The 'send' function in options.go recieves three variables

**Review by Fofichan — APPROVED** [review]
todo god

**Review by dvher — APPROVED** [review]
todo bien

**Review by benja-iag — APPROVED** [review]

**Pull request merged**